core-services/release-controller/_releases/release-ocp-4.15-arm64.json

```diff
@@ -106,6 +106,20 @@
       "prowJob": {
         "name": "periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-gcp-ovn-arm64"
        }
+    },
+    "gcp-ovn-upgrade": {
```

Comment:
I see you setting these for this [ARM 4.15 nightly stream][1].  ARM also has [preview][2] and [named 4.y.z][3] streams:

```console
$ ls core-services/release-controller/_releases | grep '\(4[.]15\|preview\|stable\)-arm'
release-ocp-4.15-arm64.json
release-ocp-4-dev-preview-arm64.json
release-ocp-4.y-stable-arm64.json
```

It looks like the 4.15 nightly stream is currently running a release about every 3h.  Named releases are 3 or so a week, and prereleases are once or so a sprint.  You can select your target based on how much you want to spend on cloud resources vs. the latency between a change landing and your hearing about any resulting issues.

[1]: https://arm64.ocp.releases.ci.openshift.org/#4.15.0-0.nightly-arm64
[2]: https://arm64.ocp.releases.ci.openshift.org/#4-dev-preview-arm64
[3]: https://arm64.ocp.releases.ci.openshift.org/#4-stable-arm64

Reply:
yeah today we don't run these jobs on named streams for multi-arches. we will have to think about the number of jobs we run if we start adding those and maybe cut down on a few jobs .